.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/.github/workflows"
```

According to [Keeping your actions up to date with Dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot) the `directory` value should be `"/"`:

```suggestion
    directory: "/"
```

---

I think that's a mistake in the docs, I have successfully used it as given and opened a PR at https://github.com/github/docs/pull/28103 about this but either way is fine, using the directory as "/" works fine as well.
P.S.: OK you are right, / is the official value, my PR got closed with the explanation that `.github/workflows` is found automatically.